lib/challenge.js

```diff
@@ -19,7 +19,16 @@
   // - park
 
 // Answer Starts Here
-
+'webCal'
+Description: An app to keep track of things via your browser.
+Every 'webCal' has (abstraction):
+  - login
+  - dates (year-month-day)
+  - color codes
+Every 'webCal' can (properties/behaviors):
+  - create/edit event
+  - delete event
+  - invite others to events
```

Comment:
This is good modeling. Trouble is that this is plain-English text in a code file and it's not commented out. As a result, you were unable to run the included tests on your potential solutions, which would have caught and helped you find errors that would keep your code from running.

This question asked for three such models, and here you've got one. 1/2